exampleSite/config.toml

```diff
@@ -127,11 +127,8 @@ paginate = 10
 # Enable or disable top bar with social icons
```

Comment:
@ [line 85](https://github.com/devcows/hugo-universal-theme/pull/246/files#diff-991d2a2fe208cdee83955ad6e9a323a7R85) you should update the comment about the default theme color (now turquoise instead of light-blue)

Reply:
Done